tailwind.config.js

```diff
@@ -68,56 +68,52 @@ function getColors(color, shades) {
 }
 
 module.exports = {
-  purge: false,
-  theme: {
-    colors: {
-      ...colors,
-      primary: colorPrimary, 
-      secondary: colorSecondary, 
-      pink: colorPink,
-      yellow: colorYellow,
-      gray: colorGray,
-      'code-editor': {
-        dark: '#252f3f',
-        'dark-highlight': '#3e4d64',
-      },
-      dark: '#1B1130',
-      'cool-gray':'#9C96A8',
-      hub: '#252f3f',
-      slack: '#371038',
-      'mac-window': {
-        'close': '#ff5f56',
-        'minimize': '#ffbd2e',
-        'maximize': '#28c93f',
-      },
-    },
+  content: ["./pages/**/*.{js,jsx}",
+  "./components/**/*.{js,jsx}",],
+  theme:{
     fontWeight: {
       'extra-light': 200,
       light: 300,
       regular: 400,
       medium: 500,
-      'semibold': 600,
+      semibold: 600,
       bold: 700,
-      'extrabold': 800,
+      extrabold: 800,
       black: 900
     },
-    animations: {
-      blink: {
-        from: {
-          opacity: 1,
-        },
-        '50%': {
-          opacity: 0,
-        },
-        to: {
-          opacity: 1,
-        },
-      },
+    fontFamily: {
+      'sans': ['Inter', ...defaultTheme.fontFamily.sans], 
+      'heading': ['Work Sans', ...defaultTheme.fontFamily.sans],
+      'body': ['Inter', ...defaultTheme.fontFamily.sans], 
+      'mono': ['Fira Code VF', 'Fira Code', ...defaultTheme.fontFamily.mono],
     },
-    animationTimingFunction: {
-      'step-end': 'step-end',
+    minWidth: {
+      '72': '18rem',
+      '40': '10rem',
+      '12': '3rem',
     },
     extend: {
+      colors: {
```

Comment:
![Screenshot from 2022-07-26 21-54-32](https://user-images.githubusercontent.com/76521428/181062948-3ceab9ec-9046-4b6e-827f-37d18a4b7aa1.png)

@mcturco Do take a review of this file. I'm getting these warnings in production as well in development.

Reply:
https://github.com/tailwindlabs/tailwindcss/blob/master/src/public/colors.js#L281 It's done by tailwind team to inform people that these colors were renamed and in future (maybe in 4.X.X) they will remove these colors from package, now they made fallbacks. If we wanna remove that warnings we need to make sure that we don't use these colors in our codebase and we can remove them.

Reply:
Ok I fixed it.